tests/cypress/integration/reportRepair/confirmation.spec.js

```diff
@@ -1,97 +1,22 @@
 import {
+  completeJourney,
   intercept_address_search,
   intercept_availability_search,
   intercept_repair_triage,
   intercept_save_repair,
-  navigateToPageSelectRadioOptionAndContinue,
-  navigateToPageTypeInputTextAndContinue
 } from '../../support/helpers';
 
-const address = '1 Downing Street, London, SW1A 2AA';
-const repairDescription = 'Eius postea venit saepius arcessitus.'
 const phoneNumber = '07512345678';
 const email = 'harrypotter@hogwarts.com';
 const repairID = '1234ABC'
 
-function completeJourney(contactPhone = false) {
-
-  cy.visit('http://localhost:3000/report-repair/');
-
-  navigateToPageSelectRadioOptionAndContinue({
-    page: 'priority-list',
-    option: 'Something else'
-  })
-
-  navigateToPageSelectRadioOptionAndContinue({
-    page: 'communal', option: 'No'
-  })
-
-  navigateToPageTypeInputTextAndContinue({
-    page: 'postcode', inputText: 'SW1A 2AA'
-  })
-
-  cy.get('[data-cy=address]', {timeout: 10000}).then(() => {
-    cy.get('select').select(address)
-    cy.get('button').click();
-  });
-
-  navigateToPageSelectRadioOptionAndContinue({
-    page: 'repair-location', option: 'Kitchen'
-  })
-
-  navigateToPageSelectRadioOptionAndContinue({
-    page: 'repair-problem',
-    option: 'Cupboards, including damaged cupboard doors'
-  })
-
-  navigateToPageSelectRadioOptionAndContinue({
-    page: 'repair-problem-best-description', option: 'Hanging door'
-  })
-
-  cy.get('[data-cy=repair-description]', {timeout: 10000}).then(() => {
-    cy.get('textarea').type(repairDescription);
-    cy.get('button').contains('Continue').click();
-  });
-
-  cy.get('[data-cy=repair-image-upload]', {timeout: 10000}).then(() => {
-    cy.get('button').contains('Continue').click();
-  });
-
-  navigateToPageTypeInputTextAndContinue({
-    page: 'contact-person',
-    inputText: phoneNumber
-  })
-
-  cy.get('[data-cy=contact-details]', {timeout: 10000}).then(() => {
-    if (contactPhone) {
-      cy.contains('Text message (recommended)').click().then(() => {
-        cy.get('input#contactDetails-text').type(phoneNumber);
-      })
-    } else {
-      cy.contains('Email').click().then(() => {
-        cy.get('input#contactDetails-email').type(email);
-      })
-    }
-    cy.get('button').click();
-  });
-
-  cy.get('[data-cy=repair-availability]', {timeout: 10000}).then(() => {
-    cy.get('[data-cy=availability-slot-0-0]').click();
-    cy.get('button').click();
-  });
-
-  cy.get('[data-cy=summary]', {timeout: 10000}).then(() => {
-    cy.get('button').click();
-  });
-}
-
 describe('confirmation', () => {
   before(() => {
     intercept_availability_search();
     intercept_address_search();
     intercept_repair_triage();
     intercept_save_repair(repairID);
-    completeJourney();
+    completeJourney(false, email);
```

Comment:
Should we name this slightly differently? I'm thinking that `completeJourney()` and then passing `false`, `email`, doesn't immediately make sense when first reading it. 
Maybe something like `completeJourneyUsingPhone()`
Purely a suggestion though.

Reply:
I agree with Renny's suggestion, it seems more meaningful.

Reply:
Good suggestion!

Have implemented now.